packages/blade/.storybook/react-native/storybook.requires.js

```diff
@@ -22,7 +22,11 @@ if (parameters) {
 argsEnhancers.forEach((enhancer) => addArgsEnhancer(enhancer));
 
 const getStories = () => {
-  return [require('../../src/components/Button/Button.stories.tsx')];
+  return [
+    require('../../src/components/Button/Button.stories.tsx'),
+    require('../../src/components/Typography/BaseText.stories.tsx'),
+    require('../../src/storybook-recipes/SimpleCardWithTokens/CardWithTokens.stories.tsx'),
```

Comment:
This crashes on react native since it internally uses html elements. Can we somehow ignore this for native?